readme.md

```diff
@@ -27,17 +27,17 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your GitHub username and copy the links to the `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://denysherych.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://denysherych.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="946" alt="image" src="https://github.com/user-attachments/assets/bd890b3c-718a-4686-9ac2-037855cf3b90" />